services/app-api/forms/mcpar.json

```diff
@@ -212,6 +213,54 @@
               }
             ]
           }
+        },
+        {
+          "name": "Add In Lieu of Services (optional)",
+          "path": "/mcpar/program-information/add-in-lieu-of-services",
+          "pageType": "standard",
+          "verbiage": {
+            "intro": {
+              "section": "Section A: Program Information",
+              "subsection": "In Lieu of Services and Settings Offered",
+              "info": [
+                {
+                  "type": "html",
```

Comment:
This is a kind of small thing, but maybe we can structure this content as `p` tags so there is consistent markup in the separate blocks of text (instead of `div` and `span` respectively)...Would require using the `"children": []` syntax in the second chunk to manage the external link, but the end result will look the same, just with slightly cleaner markup. Not a dealbreaker for this feature, obv!

**Before:**
<img width="688" alt="Screenshot 2024-05-06 at 10 07 25 AM" src="https://github.com/Enterprise-CMCS/macpro-mdct-mcr/assets/522911/d450958a-4f22-4c9d-bcf6-604ab7104fd5">

**After:**
<img width="682" alt="Screenshot 2024-05-06 at 10 07 50 AM" src="https://github.com/Enterprise-CMCS/macpro-mdct-mcr/assets/522911/cf42ec82-aae8-4ebc-a743-4b42550b63b8">

Reply:
Good callout! Just updated that 💯